tools/lofi/probabilistic_inundation.py

```diff
@@ -500,104 +474,80 @@ def inundate_probabilistic(
     features = ensembles.coords['feature_id']
 
     # For each feature in the provided ensembles
-    with ThreadPoolExecutor(max_workers=1) as executor:
-
-        # Get max streamflow for every feature up to forecast time
-        if aggregate_forecasts == "max_to_forecast":
 
-            sel_forecast = ensembles.sel({'time': slice(reference_time, forecast_time)}).max('time')[
-                'streamflow'
-            ]
-
-        # Timeslice representing the max streamflow for any feature id in time up to forecast time
-        elif aggregate_forecasts == "timeslice_max_of_any_feature_id":
-
-            sel_forecast = ensembles['streamflow'].isel(
-                {
-                    'time': ensembles['streamflow']
-                    .sel({'time': slice(reference_time, forecast_time)})
-                    .max(['feature_id', 'member'], skipna=True)
-                    .argmax()
-                }
-            )
+    # Get max streamflow for every feature up to forecast time
+    if aggregate_forecasts == "max_to_forecast":
+        sel_forecast = ensembles.sel({'time': slice(reference_time, forecast_time)}).max('time')['streamflow']
+
+    # Timeslice representing the max streamflow for any feature id in time up to forecast time
+    elif aggregate_forecasts == "timeslice_max_of_any_feature_id":
+        sel_forecast = ensembles['streamflow'].isel(
+            {
+                'time': ensembles['streamflow']
+                .sel({'time': slice(reference_time, forecast_time)})
+                .max(['feature_id', 'member'], skipna=True)
+                .argmax()
+            }
+        )
 
-        # Timeslice representing the max sum of streamflow for all feature ids in time up to forecast time
-        elif aggregate_forecasts == "timeslice_max_sum":
+    # Timeslice representing the max sum of streamflow for all feature ids in time up to forecast time
+    elif aggregate_forecasts == "timeslice_max_sum":
+        sel_forecast = ensembles['streamflow'].isel(
+            {
+                'time': ensembles['streamflow']
+                .sel({'time': slice(reference_time, forecast_time)})
+                .sum(['feature_id', 'member'], skipna=True)
+                .argmax()
+            }
+        )
 
-            sel_forecast = ensembles['streamflow'].isel(
-                {
-                    'time': ensembles['streamflow']
-                    .sel({'time': slice(reference_time, forecast_time)})
-                    .sum(['feature_id', 'member'], skipna=True)
-                    .argmax()
-                }
-            )
+    # Timeslice at forecast time
+    else:
+        sel_forecast = ensembles.sel({'time': forecast_time})['streamflow']
 
-        # Timeslice at forecast time
-        else:
+    # Generate streamflow likelihoods for each feature
+    for feat in features:
+        feat = feat if isinstance(feat, int) else int(feat)
```

Comment:
This could probably be best done by `feat = int(feat)`. No need to call isinstance. Another option would be to do `for feat in map(int, features)`

Reply:
I made this change and tested it locally and in our lambda run.